lib/legitbot/ias.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Legitbot # :nodoc:
+  # https://integralads.com/ias-privacy-data-management/policies/site-indexing-policy/
+  class Ias < BotMatch
+    # @fetch:url https://integralads.com/policy-docs/iasbot.json
+    # @fetch:jsonpath $.publicIPs[*].ipv4
+    ip_ranges %w[
+      3.217.168.199
+      18.214.43.70
+      3.226.51.67
```

Comment:
Could you please sort the list? Then the next automatic diff will be cleaner.

You don't have to do it manually. You can just remove a line, run `rubocop -A` and the custom rule will take care of soring the list properly.

Reply:
Done! Just amended the commit and forced push the branch.